daml-model/daml/Tests/LoanManagementTest.daml

```diff
@@ -0,0 +1,81 @@
+module Tests.LoanManagementTest where
+
+import Trigger.LoanManagement
+import Daml.Trigger.Assert (toACS, testRule, flattenCommands, assertExerciseCmd)
+import Daml.Script
+import Tests.PartyAlloc
+import Model.Balance
+import Model.LoanConfig
+import qualified DA.Map as M
+import Daml.Trigger (Command)
+
+
+setup = do
+    proposalTime <- getTime
+    banks@Banks{..} <- actionPartyAllocation
+    schedulerAndAssembler@ServiceParties{..} <- servicePartyAllocation
+    users@UserParties{..} <- userPartyAllocation
+
+    let approvers = [cb, bank11, bank12, bank21, bank22]
+    pure (banks, schedulerAndAssembler, users)
+
+
+createLoanWithMaxLoanPerTransaction : Script ()
+createLoanWithMaxLoanPerTransaction = do
+    (Banks{..}, ServiceParties{..}, users@UserParties{..}) <- setup
+    aliceBalanceCid <- submit bank11 do 
+        createCmd Balance with 
+            provider = bank11
+            owner = Some users.alice
+            iban = "CH123"
+            currency = "USD"
+            amount = 0.0
+
+    loanConfigCid <- submit bank11 $ do createCmd
+        LoanConfig with
+            provider = bank11
+            accountHolder = users.alice
+            account = "CH123"
+            activeLoan = 0.0
+            maxLoanPerTransaction = 1000.0
+            maxLoanThreshold = 10000.0
+            paybackThreshold = None
+
+    commands <- getTriggerResults bank11 loanConfigCid aliceBalanceCid
+
+    verifyCommandsCount commands 1
+    verifyLoanAmount loanConfigCid commands (Some 1000.0)
+    -- verifyBalance aliceBalanceCid commands 1000.0
```

Comment:
This is just a reminder to delete commented lines when ready (ie. when you publish a PR from the draft) :)

Reply:
Is verifyBalance unused?